.github/workflows/validate.yml

```diff
@@ -11,11 +11,16 @@ concurrency:
   group: ${{ github.ref }}-${{ github.workflow }}
   cancel-in-progress: true
 
+# Note: This workflow file contains the required job "Validate post job". We are using path filtering
+# here to ignore PRs which only change documentation. This can cause a problem, see the workflow file
+# "validate.patch.yml" for a description of the problem and the patch provided in that file.
```

Comment:
I'm not particularly happy with the name of the file `validate.patch.yml` and would suggest trying something more descriptive. Without thinking too much about it, the first thing that comes to mind is `validate.backup.yml` but maybe there are better options… Another idea: `validate.skip.yml`.

Reply:
I changed the name of the file to `validate.skip.yml`